spryker/usr/local/share/spryker/spryker_functions.sh

```diff
@@ -119,6 +119,7 @@ do_spryker_install() {
     do_spryker_import_demodata
     do_spryker_product_label_relations_update
     do_spryker_setup_search
+    do_spryker_app_permissions
     do_spryker_run_collectors
```

Comment:
Would mean waiting up to a minute to see something on Yves after the cron container is deployed.